manifest.schema.yaml

```diff
@@ -98,6 +98,9 @@
         "filter": {
           "type": "string"
         },
+        "schedule": {
```

Comment:
Notice that connection is still required, according to the `required` below. Maybe use `oneOf` like above?

Reply:
discussed. see ENG-862.